src/patient-registration/patient-registration.component.tsx

```diff
@@ -141,7 +199,11 @@ export const PatientRegistration: React.FC = () => {
                 <DummyDataInput setValues={props.setValues} />
               ) : null}
             </div>
-            <DemographicsSection setFieldValue={props.setFieldValue} values={props.values} />
+            <DemographicsSection
+              setFieldValue={props.setFieldValue}
+              values={props.values}
+              identifierTypes={identifierTypes}
+            />
             <ContactInfoSection />
```

Comment:
Are identifiers demographic information? I think it would be nicer to just have
```jsx
  <DemographicsSection setFieldValue={props.setFieldValue} values={props.values} />
  <IdentifiersSection .......  />
  <ContactInfoSection />
```

Reply:
But that's maybe a design decision— @nicholasjhill ?

Reply:
Ping @nicholasjhill  @nickjhill14 ? Or who's doing design on this, anyway?

Reply:
@rbuisson also suggested to move the identifiers to their own section as you suggested above.

Reply:
Okay! Are you going to make that change now, or should we put this PR in as is?

Reply:
Done!